ci-operator/config/redhat-cne/cloud-event-proxy/redhat-cne-cloud-event-proxy-release-4.12.yaml

```diff
@@ -69,7 +69,7 @@ tests:
         fi
         popd
         export CNE_IMG=${IMAGE_CLOUD_EVENT_PROXY}
-        export CONSUMER_IMG=quay.io/openshift/origin-cloud-event-proxy:4.18
+        export CONSUMER_IMG=quay.io/redhat-cne/cloud-event-consumer:4.18
```

Comment:
It is 4.18 because its the latest with v1 events?

Reply:
yes